src/vterm/vterm.h

```diff
@@ -219,7 +220,7 @@ typedef enum {
 
 typedef struct {
   const char *str;
-  size_t      len : 30;
+  int         len : 30;
```

Comment:
Can you mention somewhere why this change is necessary?

Reply:
The tests fail with `test/unit/testutil.lua:276: invalid value near ':' at line 18'` otherwise.

Reply:
Reverted it back to size_t by removing the bitfield in tests.